recipes/gitsome/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: gitsome
+  version: {{ version }}
+
+source:
+  fn: gitsome-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/gitsome/gitsome-{{ version }}.tar.gz
+  md5: 85217683d7c4c08fd2a9d3b415c640c9
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - gh = gitsome.main_cli:cli
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpydoc >=0.5
+    - ply >=3.4
+    - prompt_toolkit >=0.52
```

Comment:
This might be ok.  Currently I only see a minor cosmetic issue on the scroll bar completion menu.  There's a change that Jonathan Slenders suggested for [SAWS](https://github.com/donnemartin/saws) when we upgraded that to prompt_toolkit 1.0.0, I haven't determined yet if something similar needs to change for gitsome/xonsh:

https://github.com/donnemartin/saws/pull/83/files#diff-28f72dcbe31650a626606c02b3f9d21cR463